skops/io/_general.py

```diff
@@ -242,6 +242,30 @@ def object_get_instance(state, src):
     return instance
 
 
+def method_get_state(obj: Any, save_state: SaveState):
+    # This method is used to persist bound methods, which are
+    # dependent on a specific instance of an object.
+    # It stores the state of the object the method is bound to,
+    # and prepares both to be persisted.
+    res = {
+        "__class__": obj.__class__.__name__,
+        "__module__": get_module(obj),
+        "__loader__": "method_get_instance",
+        "content": {
+            "func": obj.__func__.__name__,
+            "obj": get_state(obj.__self__, save_state),
+        },
+    }
+
+    return res
+
+
+def method_get_instance(state, src):
+    loaded_obj = object_get_instance(state["content"]["obj"], src)
```

Comment:
```suggestion
    loaded_obj = get_instance(state["content"]["obj"], src)
```

Sorry, I missed this earlier, but I think this should be just plain `get_instance`, right? Analogous change to the one discussed `method_get_state` in an earlier review. Apart from that this LGTM.

I would like @adrinjalali to also take a look since this is very important to get right.

Reply:
I think you’re right, happy to commit this as long as the other maintainers agree